tools/diff/diff.xml

```diff
@@ -12,40 +13,92 @@
     </stdio>
 
     <command ><![CDATA[
-        diff -u '$input1' '$input2' > '$output1'
+        diff -u '$input1' '$input2' > '$diff_file';
+        #if $generate_report:
+            sed -e 's/@@title@@/Diff report for '$input1.element_identifier' and '$input2.element_identifier'/g'
```

Comment:
lets see if this passes now, are you not closing the outer `'` here?

Reply:
I thought the same, but... It works and passes the tests :)